Assets/Scripts/UI/InGame/Graphics/ResolutionSwitcher.cs

```diff
@@ -72,11 +65,11 @@ public void LoadPlayerPrefs()
 			DropDownLabel.text = resolution;
 			if(screen_mode == 1)
```

Comment:
`if(screen_mode == 1) Toggle.isOn = true else Toggle.isOn = false` can be made shorter
`Toggle.isOn = screen_mode == 1`